src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -15,7 +15,7 @@ import {
 import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useSaveAnonChatHook = () => {
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refreshWithId` function is now being used in the `useSaveAnonChatHook`. Ensure that this function has been properly defined and tested in the `useLocalDatabaseHook` to avoid any potential issues.
```diff
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->